pytest.ini

```diff
@@ -4,6 +4,7 @@ addopts=
 	--doctest-modules
 	--doctest-glob=pkg_resources/api_tests.txt
 	-r sxX
+	--color=yes
```

Comment:
This project attempts to avoid proliferation of non-default options or to apply "preferred settings" across projects in jaraco/skeleton. In any case, this change is seemingly unrelated to the described change, so I'm going to omit it for now.

```suggestion
```